protocol_driver_double_barrel.cc

```diff
@@ -13,38 +13,35 @@
 // limitations under the License.
 
 #include "protocol_driver_double_barrel.h"
-//#include "protocol_driver_allocator.h"
+
+#include "protocol_driver_allocator.h"
 #include "absl/base/internal/sysinfo.h"
 #include "distbench_utils.h"
 #include "glog/logging.h"
 #include "google/protobuf/repeated_field.h"
 
 namespace distbench {
 
-absl::StatusOr<std::unique_ptr<ProtocolDriver>> AllocateProtocolDriver(
-    ProtocolDriverOptions opts, int* port, int tree_depth = 0);
-
 // ProtocolDriver ===============================================
 ProtocolDriverDoubleBarrel::ProtocolDriverDoubleBarrel(int tree_depth) {
   tree_depth_ = tree_depth;
 }
 
+typedef google::protobuf::RepeatedPtrField<NamedSetting> ServerSetting;
```

Comment:
c++ people prefer using statement to typedef, since it avoids certain ambiguities

Reply:
Fixed